product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/01_connecting_your_own_cloud.mdx

```diff
@@ -88,8 +78,17 @@ Perform the following steps:
    biganimal setup-csp
    ```
   
-1. The command checks for cloud account readiness and displays the results. If requirements are not met and you need more information about the requirements, see [Preparing your Azure subscription](../preparing_cloud_account/01_preparing_azure/#configure-your-azure-subscription) or [Preparing your AWS account](../preparing_cloud_account/02_preparing_aws/#configure-your-aws-account). 
-
+1. The command checks for cloud account readiness and displays the results. You might need to manually increase resource limits and perform configurations in your cloud, if either of the following readiness checks are not met: 
+    * **If connecting to Azure**:
+       - Are the necessary Azure resource providers registered for your subscription?
+       - Is there a restriction on SKUs for the standard Esv3 family and standard D2_v4 VM size?
+       - Is the limit on the number of vCPU and public IP addresses in your region sufficient for your clusters?
+    * **If connecting to AWS**:
+       - Is the AWS CLI configured to access your AWS account?
+       - Is the limit on the number of vCPUs and Network Load Balancers (NLBs) in your region sufficient for your clusters?
+    
```

Comment:
this new line isn't rendering properly in the netlify app

![image](https://user-images.githubusercontent.com/412325/167664739-76f72d21-a16a-4bf3-b5ea-a31caeb513d3.png)

not sure why

Reply:
I'm not seeing that issue: 
![image](https://user-images.githubusercontent.com/83650384/167687915-17faa7a1-85f5-4843-a0ae-9e9eec353f6b.png)